src/session/session_api.c

```diff
@@ -826,6 +826,44 @@ __session_alter_readonly(WT_SESSION *wt_session, const char *uri, const char *co
     API_END_RET(session, ret);
 }
 
+/*
+ * __wt_session_count --
+ *     WT_SESSION->count method.
+ */
+int
+__wt_session_count(WT_SESSION *wt_session, const char *uri, int64_t *count)
+{
+    WT_DECL_RET;
+
+    WT_UNUSED(uri);
+    WT_UNUSED(wt_session);
+
+    ret = 0;
```

Comment:
Don't need ret = 0 as DECL_RET does that for you.

Reply:
Apparently this is required to workaround s_void, without adding it to the false positives list.

Reply:
As discussed offline, we're keeping this as-is due to `s_void` complaining:
```
####################### MESSAGE ############################
s_all run of: "./s_void" resulted in:
    ==============================================
    src/session/session_api.c:
        int __wt_session_count
    Function could return void instead of int.
    Add false positives to the list in dist/s_void.
    ==============================================
```

This is bound to change as we implement this function anyways.